synapse/events/spamcheck.py

```diff
@@ -603,14 +679,21 @@ async def user_may_publish_room(
                 if res is True or res is self.NOT_SPAM:
                     continue
                 elif res is False:
-                    return synapse.api.errors.Codes.FORBIDDEN
+                    return (synapse.api.errors.Codes.FORBIDDEN, {})
```

Comment:
In Python when returning multiple values the parentheses are redundant (which can cause some warnings in IDEs such as PyCharm), so I'd recommend removing them (same with other return statements with multiple return values).